README.md

```diff
@@ -208,7 +208,7 @@ Required Ruby version: > 2.4. Ruby version 3 is also supported.
 
 Refer to the following sections for a proposed method for specific operating systems.
 
-The recommended installation method is `rvm` for systems with "bash-like" shell (Linux, Macos, Windows with cygwin, etc...).
+The recommended installation method is `rvm` for systems with "bash-like" shell (Linux, MacOS, Windows with cygwin, etc...).
```

Comment:
```suggestion
The recommended installation method is `rvm` for systems with "bash-like" shell (Linux, macOS, Windows with cygwin, etc...).
```

Reply:
I agree. I made another commit for this and pushed it. Can I get that added to this pull request or do I make another pull request?